src/libuvc/ctrl.cpp

```diff
@@ -66,7 +66,7 @@ int uvc_get_ctrl_len(uvc_device_handle_t *devh, uint8_t unit, uint8_t ctrl) {
     devh->usb_devh,
     REQ_TYPE_GET, UVC_GET_LEN,
     ctrl << 8,
-    unit << 8 | devh->info->ctrl_if.bInterfaceNumber,		// XXX saki
+    unit << 8 | devh->info->ctrl_if.bInterfaceNumber,        // XXX saki
```

Comment:
The libuvc should probably be excluded, at least temporally. So then if we get a fix from a 3rd party to libuvc library, it will be much easier to compare and merge.